src/main/drivers/pwm_output.c

```diff
@@ -280,3 +281,28 @@ void servoInit(const servoConfig_t *servoConfig)
 }
 
 #endif
+
+#ifdef BRUSHED_ESC_AUTODETECT
+uint8_t hardwareMotorType = MOTOR_UNKNOWN;
```

Comment:
Strictly speaking, this is a type mismatch. This is not a type "uint8_t" variable. It is a type "HardwareMotorTypes_e", which really should be named "HardwareMotorTypes_et" since it is a typedef-ed enum. I.e. an "enum type". But we have these constructs and type mismatches all over the place... And C allows it, so never mind. Just a comment.